docs/source/index.rst

```diff
@@ -52,35 +51,26 @@ Here is a sample DNP3 driver configuration file:
 
-The driver's registry configuration can be a CSV file with one row for each point or a JSON file structured as a list
-of objects (one object for each point). Either the columns (CSV) or keys (JSON) are required for each point.
+All valid data types and formats in DNP3 are identified by group and variation numbers. When a DNP3 outstation
+transmits a message containing response data, the message identifies the group number and variation of every value within
+the message. The group-variation pair numbers provide sufficient information for the receiver to parse and
+properly interpret the data. DNP3’s basic documentation contains a `table <https://docs.stepfunc.io/dnp3/0.9.0/dotnet/namespacednp3.html#a467a3b6f7d543e90374b39c8088cadfbaff335165a793b52dafbd928a8864f607>`_
+of valid groups and their variations.
 
-As a reminder, When a DNP3 outstation transmits a message containing response data, the message identifies the group number and variation
-of every value within the message. Group and variation numbers are also assigned for counters, binary inputs, controls and
-analog outputs. In fact, all valid data types and formats in DNP3 are identified by group and variation numbers. Defining the
-allowable groups and variations helps DNP3 assure interoperability between devices. DNP3's basic documentation (i.e., `dnp3.Variation
-<https://docs.stepfunc.io/dnp3/0.9.0/dotnet/namespacednp3.html#a467a3b6f7d543e90374b39c8088cadfbaff335165a793b52dafbd928a8864f607>`_) contains a
-library of valid groups and their variations.
-
-In addition, when data from an index is transmitted across the wire, the sender must suitably encode the information to enable a receiving
-device to parse and properly interpret this data. The bits and bytes for each index appearing in the message are called an object.
-That is, objects in the message are the encoded representation of the data from a point, or other structure, and the object format
-depends upon which group and variation number are chosen
-
-Point definitions in the DNP3 driver’s registry (shown as CSV) should look similar to the following:
+The driver's registry configuration file specifies information related to each point on the device, in a CSV or JSON file.
+More detailed information of driver registry files may be found `here <https://volttron.readthedocs.io/en/main/agent-framework/driver-framework/platform-driver/platform-driver.html?highlight=driver_config#driver-configuration-file>`_.
+The driver’s registry configuration must contain the following items for each point:
 
     - **Volttron Point Name** - The name used by the VOLTTRON platform and agents to refer to the point.
     - **Group** - The point's DNP3 group number.
-    - **Variation** - The permit negotiated exchange of data formatted, i.e., data type.
-    - **Index** - The point's index number within its DNP3 data type (which is derived from its DNP3 group number).
+    - **Variation** - The point's DNP3 Variation number.
```

Comment:
If variation is capitalized, group should be as well (I would say both probably shouldn't, but that depends on how the standard refers to them).